Makefile

```diff
@@ -66,19 +61,25 @@ mem.png: mem.dat mem.gp
 	gnuplot mem.gp
 	open mem.png
 
-goxc:
+goxc: dev
 	rm -rf root/
 	mkdir -p root/usr/share/man/man1/ root/etc/bash_completion.d/ root/usr/share/zsh/vendor-completions/
 	cp man/aptly.1 root/usr/share/man/man1
 	cp completion.d/aptly root/etc/bash_completion.d/
 	cp completion.d/_aptly root/usr/share/zsh/vendor-completions/
 	gzip root/usr/share/man/man1/aptly.1
-	goxc -pv=$(VERSION) -max-processors=4 $(GOXC_OPTS)
+	goxc -pv=$(VERSION) -max-processors=2 $(GOXC_OPTS)
```

Comment:
Is there a reason this is hardcoded to 2 rather than get the cpu count from `nproc`?

Reply:
This is more for local dev, I think `go` does parallel builds on its own, so combined parallelization of goxc + go almost freezes my laptop during the builds. Travis I think doesn't have many procs as well ;)